### PR TITLE
#285 styling ring menu

### PR DESCRIPTION
**Pull request by hopetambala**
### Description of proposed changes
Styling of Ring Menu so that there's a transition upon entrance.
Aligning icons inside icon container so that there's a cleaner layout
Changing the text color on the Node Groups in Dark theme to align with the theme

### Checklist
_Put an `x` in the boxes that apply. _
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [x] I have added or updated necessary documentation (if appropriate)

### Additional comments
Is there anything else you want us to know?

### CODE REVIEW

File: src/features/visualization/d3-viz/setup-annotations.js [hunk 1]
```diff
@@ -39,8 +39,11 @@ const setupAnnotations = ({packedData, annotationRoot}) =>{
   .append('text')
     .classed('svg-icon', true)
     .classed('nag', true)
-	 .style('font-size', (d) => (3 * d.height * fontScale) + "%")
-    .text('\uf06a');//font-awesome
+	 .style('font-size', (d) => (4 * d.height * fontScale) + "%")
+	 .text('\uf05a');//font-awesome information icon circle
```
Comment (rashley-iqt) [inline src/features/visualization/d3-viz/setup-annotations.js]: Please revert this change to the exclamation point. The info icon isn't considered urgent enough

Comment (hopetambala) [reply src/features/visualization/d3-viz/setup-annotations.js]: Reverted, but I did increase the icon size
